plinic/src/components/input/Input.jsx

```diff
@@ -0,0 +1,140 @@
+import React, { useState } from 'react';
+import styled, { css } from 'styled-components';
+
+function Input({ usedFor }) {
+  const [userInput, setUserInput] = useState('');
+  const [userSubmit, setUserSubmit] = useState('');
+
+  const placeholder = () => {
```

Comment:
혹시 여기 파라미터로 usedFor를 넣어야 하는 건 아닐까요? 현재 상태에서 코드가 잘 동작하나요?
```suggestion
const placeholder = (usedFor) => {
```

Reply:
동작은 하지만 넣는 게 좋아 보여서 수정하겠습니다.